imagecraft/models/volume.py

```diff
@@ -55,13 +56,47 @@
     "and hyphens, and may not begin or end with a hyphen."
 )
 
+SIZE_INVALID_MSG = "size must be expressed in bytes, optionally with M or G unit."
+
 StructureName = typing.Annotated[
     str,
     StringConstraints(
         max_length=GPT_NAME_MAX_LENGTH, pattern=STRUCTURE_NAME_COMPILED_REGEX
     ),
 ]
 
+
+def _validate_structure_size(value: str) -> str:
+    """Validate and convert the structure size.
+
+    The Volume specification does not respect the IEC 80000-13 Standard:
```

Comment:
Maybe use different wording here? Instead of mentioning a standard we don't follow, we could just say what the expected format is, and let the validator do its magic to produce the final value. Personally I would just resolve the value directly instead of converting units.

Reply:
Done. Let me know what you think.